packages/suite-web/e2e/tests/onboarding/t2t1-create-wallet.test.ts

```diff
@@ -16,8 +14,8 @@ describe('Onboarding - create wallet', () => {
         // in production, and we locally bumped emulator version, we would get into a screen saying "update your firmware" and the test would fail.
         cy.task('startEmu', { wipe: true, model: 'T2T1', version: '2-main' });
 
-        // TODO: compromised device workaround, refactor into more stable solution
-        cy.contains('Back').click();
+        // firmware revision check will fail with 2-main version, but we just need to get through onboarding, so it's fine to just dismiss it
+        cy.getTestElement('@device-compromised/back-button').click();
```

Comment:
THis is the only case where clicking on Back button remains necessary, as we use `2-main`
But it's better to use `data-testid` which was introduced in #15031